framework/include/media/MediaRecorder.h

```diff
@@ -205,7 +214,7 @@ class MediaRecorder
 	/**
 	 * @brief MediaRecorder operator==
 	 * @details @b #include <media/MediaRecorder.h>
-	 * This function is a synchronous api
+	 * This function is a synchronous API
```

Comment:
Same.
All the corrections in this file can be added to a seperate commit.

Reply:
Okay, thanks. I'll do so from now on.